com.unity.ml-agents/Runtime/Communicator/RpcCommunicator.cs

```diff
@@ -537,6 +537,43 @@ public void UnregisterSideChannel(SideChannel sideChannel)
             }
         }
 
+        /// <summary>
```

Comment:
Do we expect the implementation of these two methods to be different for different ICommunicator implementations?

Reply:
The implementation will depend on how the SideChannels are stored by the ICommunicator implementation. There's no guarantee that another implementation would use a UUID->SideChannel dictionary.

We could make this an internal static method in SideChannel.cs if you think it's likely to be reusable.